contracts/0.8.9/oracle/AccountingOracle.sol

```diff
@@ -714,17 +716,27 @@ contract AccountingOracle is BaseOracle {
             if (itemType == EXTRA_DATA_TYPE_EXITED_VALIDATORS ||
                 itemType == EXTRA_DATA_TYPE_STUCK_VALIDATORS
             ) {
-                _processExtraDataItem(data, iter);
+                uint256 nodeOpsProcessed = _processExtraDataItem(data, iter);
+
+                if (nodeOpsProcessed > maxNodeOperatorsPerItem) {
+                    maxNodeOperatorsPerItem = nodeOpsProcessed;
+                    maxNodeOperatorItemIndex = index;
+                }
             } else {
                 revert UnsupportedExtraDataType(index, itemType);
             }
 
             assert(iter.dataOffset > dataOffset);
             dataOffset = iter.dataOffset;
         }
+
+        if (maxNodeOperatorsPerItem > 0) {
+            IOracleReportSanityChecker(LOCATOR.oracleReportSanityChecker())
+                .checkNodeOperatorsPerExtraDataItemCount(maxNodeOperatorItemIndex, maxNodeOperatorsPerItem);
+        }
     }
 
-    function _processExtraDataItem(bytes calldata data, ExtraDataIterState memory iter) internal {
+    function _processExtraDataItem(bytes calldata data, ExtraDataIterState memory iter) internal returns (uint256 nodeOpsProcessed) {
```

Comment:
I'd propose changing this to `nodeOpsCount` and either removing the `nodeOpsCount` variable below or removing the name here